index.bs

```diff
@@ -947,8 +950,8 @@ ModuleStatus instances are initially created with the internal slots described i
 1. Let _translateStageEntry_ be GetStage(_entry_, "translate").
 1. If _translateStageEntry_ is *undefined*, return a promise resolved with *undefined*.
 1. If _translateStageEntry_.[[Result]] is not *undefined*, return _translateStageEntry_.[[Result]].
-1. Let _hook_ be GetMethod(_entry_.[[Loader]], @@translate).
 1. Let _p_ be the result of transforming RequestFetch(_entry_) with a fulfillment handler that, when called with argument _payload_, runs the following steps:
+  1. Let _hook_ be GetMethod(_entry_.[[Loader]], @@translate).
```

Comment:
I don't have super strong rationale either way. What you've changed it to is slightly more flexible, since you could choose to change a later hook based on the result of an earlier one. But in practice I don't think you'd want to do that, since it races badly with other concurrently loaded modules. In the end, I doubt it matters much.